packages/docs/docs/usage/layout-grid.md

```diff
@@ -0,0 +1,1142 @@
+`{ "ui:col": ["person"] }`, in a manner similar `ui:row` where the array is implicitly assumed to be the `children`
+property.
+
+In its most expanded form, the `ui:col` contains an array of `children` in the form of objects with `"name": "<field id>"`
+as the minimum required property. Any additional properties added within the object will be passed along to the
+underlying `Widget` implementation in the `ui:options` portion of the `uiSchema`. This allows for fine-tuning any
+out-of-the-box or custom widgets. Let's take a look at an example where the `Middle Name` column hides its label adding
+a margin to take up the space once occupied by the label, and is reduced in size relative to the `First Name` and
+`Last Name`:
+
+```json
+{
+  "ui:field": "LayoutGridField",
+  "ui:layoutGrid": {
+    "ui:row": {
+      "className": "row",
+      "children": [
+        {
+          "ui:row": [
+            {
+              "ui:col": {
+                "className": "col-xs-12",
+                "children": ["person"]
+              }
+            }
+          ]
+        },
+        {
+          "ui:row": [
+            {
+              "ui:col": {
+                "className": "col-xs-5",
+                "children": ["person.first"]
+              }
+            },
+            {
+              "ui:col": {
+                "className": "col-xs-2",
+                "style": {
+                  "margin-top": "25px"
+                },
+                "children": [
+                  {
+                    "name": "person.middle",
+                    "label": false
+                  }
+                ]
+              }
+            },
+            {
+              "ui:col": {
+                "className": "col-xs-5",
+                "children": ["person.last"]
+              }
+            }
+          ]
+        }
+      ]
+    }
+  },
+  "person": {
+    "ui:field": "LayoutHeaderField"
+  }
+}
+```
+
+Passing the above `uiSchema` to the RJSF `Form` you will end up with the following UI:
+
+```text
+Person Info
+  First Name*                                      Last Name*
+  +--------------------------------+  +---------+  +--------------------------------+
+  |                                |  |         |  |                                |
+  +--------------------------------+  +---------+  +--------------------------------+
+```
+
+### ui:columns
+
+The `ui:columns` "syntax" represents a group of columns in the grid. Like `ui:row` and `ui:col` this syntax will spread
+the `"className": "col-xs-4"` properties onto the underlying theme's grid implementation, as well as any other properties,
+such as `style` blocks, etc. It is short-hand notation for rendering a group of columns all with the same grid system
+properties. As shown in the above examples, the set of three `ui:col` definitions for first, middle and last names are
+the same as the single `ui:columns` definition with all three listed as children.
```

Comment:
Clarify that the passed props are arbitrary/theme-based.  The example you're referencing is a couple of examples ago.
```suggestion
The `ui:columns` "syntax" represents a group of columns in the grid. Like `ui:row` and `ui:col` this syntax will spread
the properties onto the underlying theme's grid implementation (in an earlier example, `"className": "col-xs-4"`), as well as any other properties,
such as `style` blocks, etc. It is short-hand notation for rendering a group of columns all with the same grid system
properties. As shown in the above examples, the set of three `ui:col` definitions for first, middle and last names are
the same as the single `ui:columns` definition with all three listed as children.
```